### ISSUE TITLE
Output interrupt when a "è" it's present in the file

### ISSUE THREAD

**Issue by eathtespagheti** [jaseg/lolcat#30]
Example:
textfile:
```
This is a file with a è character
```

`lolcat textfile` output:
```
This is a file with a
```

Expected output:
```
This is a file with a è character
```

**Comment by neinseg** [comment]
Hey there,

I'm having trouble reproducing your issue.

in case you're still having this issue, please update this issue with information on which platform you're using and which terminal emulator you're using. Please also attach the output of the following two commands: ```$ hexdump -C textfile``` where ```textfile``` is the textfile causing this issue, as well as ``$ locale``

Thank you for the effort.
-- jaseg

**Comment by eathtespagheti** [comment]
Yeah the issute it's still there, as mentioned on issue #21 running `lolcat -l`allow to output the file anyway, but sometimes I get a random ERRNO 2 with output `Error reading input file "textfile": Success` at the end of the normal cat output

**Comment by jaseg** [comment]
I'm still not sure what's causing your lolcat to choke on that UTF-8 multibyte sequence. I'm looking into that rn. The other issue you're describing, where lolcat outputs random error messages at the end of output is #35 and has been fixed in master.
